bases/ember.json

```
@@ -2,15 +2,15 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Ember",
   "docs": "https://docs.ember-cli-typescript.com",
-  "_version": "2.0.0",
+  "_version": "2.1.0",
```

As suggested on the main discussion, it would be helpful for us to treat this as 3.0.0 if it’s explicitly bumping the minimum TS version. However, I have not fully taken on the overall approach here. If (per the README change) it is going going to be effectively bumped with every release, we may need to reevaluate the approach. (Ember itself is also at a point where we could poeple finally look at just shipping an `@ember/tsconfig` instead, inverting this and leaving Ember in control of those bumps. I will dig in further and chat with the rest of the team and follow up!

---

Happy with whatever works for you - could bump to 4.x because that matches the LTS of embed for example too?

---

Let's just go with a clean major for now:

```suggestion
  "_version": "3.0.0",
```

Medium-term, I think we'll probably end up switching to an `@ember/tsconfig.json` so we can manage our own versioning scheme with it (that makes good sense for how we're approaching Ember's own SerVer commitments re: TS support).